e2e/menu.test.ts

```diff
@@ -0,0 +1,20 @@
+import { by, device, element, expect } from 'detox';
+
+describe('Menu', () => {
```

Comment:
You should mock the network request and return a dummy alphabet with the same structure (i.e., TypeScript type), but fake data.

Reply:
Alternatively, you can inject a `DummyConfig` that points to the api running over local host. Then you need to make sure to run the API locally before running tests. If you do this, document this requirement in the `README.md`.